src/components/PluginWallet/PluginWallet.tsx

```diff
@@ -34,8 +34,11 @@ export function ActivePlugin(props: ActivePluginProps) {
     <>
       {iconActive}
       <div className="w-full py-6 text-f-primary bg-secondary flex flex-col items-center">
-        <div className="w-4/5 px-4 py-2 text-center mas-subtitle min-[450px]:mas-title">
-          {title}
+        <div
+          className="px-4 py-2 lg:h-14 mas-title min-[0px]:text-xl sm:text-4xl
+                        min-[790px]:text-4xl lg:text-2xl xl:text-3xl min-[1330px]:text-4xl"
```

Comment:
min-[790px], 
min-[1330px]

Question: Is there a way to **not** rely on that specific values!? I mean... using at least percentage or calc in function of the screen!? 🤔

Reply:
I have simplified

Reply:
Awesome! Tks!